spec/models/room_spec.rb

```diff
@@ -0,0 +1,11 @@
+require "rails_helper"
+
+describe "Room" do
+  it "must have a name" do
+    expect(Room.new).not_to be_valid
+  end
+
+  it "must have at least one choice" do
+    expect(Room.new(name: "test")).not_to be_valid
```

Comment:
Same goes for here. If the test name is saying that it should have something, then the expectation should be that it has something and is valid with that thing.